### DISCUSSION TITLE
Button color name docs

### DISCUSSION THREAD

**Discussion by pchemguy** [sqlpage/SQLPage#409] [Q&A] [answered]
The docs for the button color attribute states "The color of the button (e.g., red, green, blue, but also **primary, warning, danger, orange, etc.**)". Orange clearly does not belong in the list. But more importantly, the associated link to tabler.io does not provide any info on the definition of color names in bold. It appears that these names are defined by Bootstrap. I might be wrong, but it would be nice if you provided appropriate reference in the docs.

**Comment by lovasoa** [comment, marked as answer]
This should be fixed by https://github.com/lovasoa/SQLpage/commit/063a626c7235817940c951aea5e666b039156b46

**Reply by lovasoa** [reply]
See the new https://sql.ophir.dev/colors.sql

**Reply by Pieter3033** [reply]
Thank you for adding this 🔥

**Reply by lovasoa** [reply]
No problem, @Pieter3033 !
